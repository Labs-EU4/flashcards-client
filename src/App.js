import React from "react";
import AppRouter from "./router";
import { Provider } from "react-redux";
import store from "./state/store";

const App = () => {
  return (
    <Provider store={store}>
      <AppRouter />
    </Provider>
  );
};

export default App;
