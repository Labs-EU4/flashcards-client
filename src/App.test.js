import React from 'react';
import * as rtl from '@testing-library/react';
import '@testing-library/jest-dom/extend-expect';
import App from './App.js'


jest.mock('axios', () => {
    return {
      get: () => Promise.resolve({ data: [{}, {}] }),
      post: () => Promise.resolve({ data: { message: 'Created new entry' } })
    }
});

afterEach(rtl.cleanup);

let wrapper;
beforeEach(() => {
  wrapper = rtl.render(<App />)
});

it('renders fine without crashing', () => {
    expect(wrapper.container).toMatchSnapshot()
});

it('renders this is home', () => {
    let queryValue = wrapper.queryByText(/Home/)
    expect(queryValue).toBeDefined()
});