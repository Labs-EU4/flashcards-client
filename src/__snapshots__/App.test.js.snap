// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders fine without crashing 1`] = `
<div>
  <main>
    <h1>
      This is Home!
    </h1>
  </main>
</div>
`;
